"Lobotomized" and "Psycotic" traits (unused)

Blogaugis opened the pull request:
Adds "Lobotomized" and "Schizophrenic" traits into the game.

Blogaugis committed:
Update scr_marine_struct.gml

Adds "Lobotomized" and "Psychotic" traits into the game.

Blogaugis commented:
Currently, no marines spawn with at least 1 new trait (Lobotomized).
Bit of testing is needed - via cheating in the Lobotomized trait onto a marine - to see if the game won't crash on negative stat numbers.

OH296 commented on scripts/scr_marine_struct/scr_marine_struct.gml:
```diff
@@ -416,6 +416,22 @@ global.trait_list = {
+		display_name : "Psychotic",
+		flavour_text : "whether due to experience or some other cause, acquired access to hidden knowledge...",
 	}
 }
```
currently neither will spawn i presume this is known , but you only mention one not spawning. Do you understand how applying the guassian randomisation works, e.g wisdom : [10,3,"max"] if not it can be explained

Blogaugis replied:
I am positive 100% about lobotomized trait, as I would have noticed at least 1 marine having negative or very low stats in wisdom and intelligence.
Psychotic, however, is not so easy to spot, hence I was not confident in my relatively quick glance at marines.

I am basing my knowledge on Gaussian randomization on trait list CM google calculator doc, which has a diagram there. Noticing that [10 (I think this is the mean);3 (1 sd, I think); "max" (I am not sure about this one)] gave negative or very low values at times, I prefer to avoid using Gaussian randomization, except for starting hireling and marine stats, but even then I tend to lower 1 sd, sometimes to just 1.

I am a bit skeptical using Gaussian randomization on most of the traits, as traits giving random stats seems odd to me, especially if it comes to true/false kind of situations - the person either knows or does not know.
However, I can see this being useful to experience traits, which are not so strictly defined as true/false, and varies from one person to another.
Lobotomite can be a good example for this - some person may suffer this injury in 1 section of the brain, other in a different one, hence the random stat affects from traits make sense.

Overall, how I understand the Gaussian randomization on traits, is that:
A Gaussian randomization runs on a trait, stats generated are saved, and applied on top of the marine stats.

OH296 replied:
Understood, The trait randomisation, is essential to the overall stat spawning in order to create true randomisation but now that is set up it is largely beyond your concern. Either way it's use is not mandatory nor required in all instances however i was just ensuring it was understood.

note that for commenting out large areas you can use /* <multiple lines */ which will save time

numbers can be augmented as they all often are subject to uses being found

OH296 commented on scripts/scr_marine_struct/scr_marine_struct.gml:
```diff
@@ -416,6 +416,22 @@ global.trait_list = {
+		display_name : "Lobotomized",
+		flavour_text : "received treatment or damage, which resulted in portion of brain missing, akin to servitors",
+	},
+	"psychotic" : { // IT IS THE BANEBLAAADE!!! - Captain Diomedes, DoW 2 Retribution
```
psychotic as term seems to be a vastly negative word for an essentially net positive trait, could a more fitting name be applied to leave the term "psychotic"open for more negative conotations?

Blogaugis replied:
Crazy seems simplistic for this, and not to the point...
Schizophrenic was declined by Etty, as it is a medical condition, which does not... make sense in 40k...
Eccentric is... poorly definable, but I guess this is where the Gaussian randomization could work wonders, let it give good and bad stats to the marine.
After that, more time thinking on this is needed.

Blogaugis committed:
Update scr_marine_struct.gml

Added 2 placeholder, commented out, options that were discussed in the comments.

Blogaugis commented:
Okay, thank You.
